Gauge/CCL.cs

```diff
@@ -0,0 +1,103 @@
+using CCL.Importer.Types;
+using CCL.Types.Components;
+using Gauge.MeshModifiers;
+using Gauge.Utils;
+using UnityEngine;
+
+using static UnityModManagerNet.UnityModManager;
+
+namespace Gauge
+{
+    internal static class CCL
+    {
+        private static ModEntry s_mod = null;
+        public static ModEntry Entry
+        {
+            get
+            {
+                if (s_mod == null)
+                {
+                    modEntries.TryFind(x => x.Info.Id == "DVCustomCarLoader", out s_mod);
```

Comment:
What's the point of this extension method instead of just calling `List<T>.Find`? The return value isn't used, so I don't see why it needed to be added.